2024/selection-process

```diff
@@ -0,0 +1,139 @@
+# Proposal for Interop 2024 Process
+
+The Interop Project is an ongoing effort to make the web more interoperable in key areas, prioritized by user and web developer needs. It’s part of the [_Web Platform Tests_](https://github.com/web-platform-tests/wpt) project, the main test suite for the web platform. And it’s facilitated by the [_Interop Team_](https://github.com/web-platform-tests/interop/blob/main/supporters.md).
+
+The goal of the Interop Project is to improve the web by making it easier to make websites and web apps that work in every browser or browser engine at the same time. There are many great ways to do so, but limited resources, so hard decisions must be made about what to prioritize.
+
+Determining what will be included in the following year’s Interop Project begins with an open call for proposals, followed by a multi-round selection process, described below. Decisions made by the consensus of the Interop Team.
+* * *
+
+## Timeline
+
+1. Call for Proposal Period — three weeks (Sep 14 – Oct 7)
+2. Continued discussion and refinement of proposals — one week (Oct 8 – Oct 14)
+3. Round 1: Disqualification — one week (Oct 15 – Oct 19)
+4. Round 2: Reduction — two weeks (Oct 20 – Nov 2)
+5. Round 3: Prioritization — three weeks (Nov 3 – Nov 30)
+6. Round 4: Decision — two weeks (Dec 1 – 14, plus time in Jan as needed)
+
+Interop 2024 and the set of accepted proposals will be announced in January 2024. 
+
+This timeline can be adjusted as needed, at any time, by consensus of the Interop Team. 
+* * *
+
+## Open Call for Focus Area and Investigation Proposals
+
+An open call for proposals will begin on a specific date, and run for three weeks.
+
+1. Anyone can submit a proposal.
+2. Proposals should be for either: 
+    A. Focus Area, or
+    B. Investigation Project
+3. Related proposals should be filed separately. For example, a group of specific features to address under “Typography” should be filed as separate issues, not one issue. This makes it easier to discuss and prioritize each separately. Bundles of proposals risk being rejected or deprioritized.
+4. Focus Area proposals must: 
+    A. Name a specific web technology. 
+        i. This can be a web technology that has not yet been implemented in any or all browser engines.
+        ii. Or this can be a web technology that has already been implemented in all browser engines, but those implementations have test failures that reveal a lack of compliance to the web standard, causing a lack of interoperability.
+    B. The feature must be defined in a sufficiently mature standards-track web specification from one the following organizations. Please provide a link to the web standard. 
+        i. [IETF](https://www.ietf.org/) (Standard Track)
+        ii. [Khronos Group](https://www.khronos.org/)
+        iii. [TC39](https://tc39.es/) (Stage 2+)
+        iv. [W3C](https://www.w3.org/) 
+        v. [WHATWG](https://whatwg.org/)
```

Comment:
Shall we add the standardization groups that define image, video and audio formats? We get proposals every year for media formats. There may be good reasons to not include them (testability being one that's come up in the past), but it seems like "not the W3C, et al" is not a good one.

Reply:
c.f. #589

Reply:
I'd prefer to not do that now, let's update ahead of Interop 2025 instead, so that this matches what the proposal template said.

Reply:
Oh, I see @jensimmons made a concrete proposal in https://github.com/web-platform-tests/interop/issues/589#issuecomment-1778208293. I'm OK with that since the work was already done. Jen, you said you created a suggested amendment, do you mean the comment itself, or do you have a suggested edit to this PR sitting as a draft somewhere?